cypress/e2e/filter/filterChips.cy.ts

```diff
@@ -292,6 +295,55 @@ describe("Case list", () => {
           .should("exist")
       })
     })
+
+    describe('Applied filter chips to "Filter applied" section', () => {
+      it("Should display the Trigger filter chip when selected", () => {
+        cy.get("#filter-button").click()
+        cy.get(".govuk-checkboxes__item").contains("Triggers").click()
+
+        // Check if the correct heading and filter label are applied
+        cy.get(".govuk-heading-s").contains("Reason").should("exist")
+        cy.get(".moj-filter__tag").contains("Triggers").should("exist")
+        cy.get(".moj-filter__tag").contains("Exceptions").should("not.exist")
+
+        // Check if the filter chip is applied to the "Filters applied" section at the top of the case list
+        cy.contains("Apply filters").click()
+        cy.get(".moj-filter-tags").contains("Triggers")
+
+        // Clears filter chip and check the checkbox is deselected
+        cy.contains("Clear filters").click()
+        cy.get("#filter-button").click()
+        cy.get(".govuk-checkboxes__item").contains("Triggers").should("not.be.checked")
+      })
+    })
+
+    it("Should display the 'Locked to me' filter chip when selected", () => {
+      cy.get("#filter-button").click()
+      cy.get(".govuk-checkboxes__item").contains("View cases allocated to me").click()
+
+      // Check if the correct heading and filter label are applied
+      cy.get(".govuk-heading-s").contains("My cases").should("exist")
+      cy.get(".moj-filter__tag").contains("Locked to me").should("exist")
+
+      // Check if the filter chip is applied to the "Filters applied" section at the top of the case list
+      cy.contains("Apply filters").click()
+      cy.get(".moj-filter__tag").contains("View cases allocated to me")
```

Comment:
likewise with a `.should("exist")` here

Reply:
Do we need it? It was kept out for simplicity, I think that if "View cases allocated to me" was not there, this would fail. Shall we put this in there?

Reply:
We added this for visibility.